source/core/types/ut_executable.tpb

```diff
@@ -96,6 +97,21 @@ create or replace type body ut_executable is
   begin
     if self.is_defined() then
       l_start_transaction_id := dbms_transaction.local_transaction_id(true);
+      
+      l_client_info := case
+                         when a_item is of(ut_test) then
+                          'Test: '
+                         else
+                          'Suite: '
+                       end || a_item.name || case
```

Comment:
I think it would be better give the name from the executed procedure:
`self.object_name||'.'||self.procedure_name`
or even only `self.procedure_name`
This makes it easier to identify the actual procedure running.